tpcds/queries-spark/q35.sql

```diff
@@ -7,19 +7,19 @@ select
   cd_marital_status,
   cd_dep_count,
   count(*) cnt1,
+  min(cd_dep_count),
   max(cd_dep_count),
-  stddev_samp(cd_dep_count),
-  stddev_samp(cd_dep_count),
+  avg(cd_dep_count),
```

Comment:
The queries in this repo are the official queries as generated by the TPC-DS query generator provided by the TPC. I don't think that we should be changing them?

Reply:
I think the query generator has a bug in this case, see https://github.com/apache/datafusion/blob/main/datafusion/core/tests/tpc-ds/35.sql as another generated version that correctly substituted the aggregations

Reply:
You are correct. I just checked the query description in the TPC-DS specification, and it does not use `stdev_sample.` I wonder if this was inadvertently checked in after some manual experimentation. Thanks for catching this!

Reply:
No worries, latest version of the tpc-ds generator also generated this query wrong for me, so I think it is some bug in the param substitution in the dsqgen executable